sopel/coretasks.py

```diff
@@ -988,6 +977,8 @@ def sasl_fail(bot, trigger):
 
 
 @module.event(events.RPL_SASLMECHS)
+@plugin.thread(False)
+@plugin.priority('low')
```

Comment:
Oh, on that particular event, since the callable will use `bot.quit`, I figured it would be better to use `low`, so it ensures that anyone trying to do something on that event, even at medium priority, wouldn't have any problem with it.